plugins/embed-optimizer/readme.txt

```diff
@@ -2,7 +2,7 @@
 
 Contributors: wordpressdotorg
 Tested up to: 6.6
```

Comment:
@westonruter Do we want to already bump this to `6.7` here and elsewhere? The next releases will be after 6.7, so would be good to already bump. We already run everything against `trunk` so I think it's reasonable to mark it as such.

Reply:
Yeah, great idea

Reply:
Done in a56c1713